snap/snapcraft.yaml

```diff
@@ -128,6 +128,12 @@ parts:
       - lib/google
       - lib/protobuf-*.pth
       - lib/protobuf-*.dist-info
+      - lib/tabulate-*.dist-info
+      - lib/tabulate.py
```

Comment:
am I very anal if it bothers me that you switched the order of dist-info and the libraries?